package.json

```diff
@@ -56,6 +56,7 @@
     "react-native-bootsplash": "^5.1.3",
     "react-native-confirmation-code-field": "^7.3.1",
     "react-native-device-info": "^8.7.1",
+    "react-native-fs": "^2.20.0",
```

Comment:
Does this need [configuration in the native code](https://github.com/itinance/react-native-fs?tab=readme-ov-file#usage-android)?

Reply:
from my understanding it doesn't. react native's automatic autolinking works with it. just from running the app, that seems to be the case